SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -156,6 +156,12 @@ void USpatialReceiver::OnRemoveEntity(Worker_RemoveEntityOp& Op)
 	RemoveActor(Op.entity_id);
 }
 
+void USpatialReceiver::UpdateShadowData(const Worker_EntityId& EntityId)
```

Comment:
```suggestion
void USpatialReceiver::UpdateShadowData(Worker_EntityId EntityId)
```
Nit: `Worker_EntityId` is just an int64, there's no benefit of passing it by const-ref.